__tests__/app.test.js

```diff
@@ -38,3 +38,25 @@ describe("GET /api/topics" , () => {
   });
 });
 
+describe("GET /api/articles/:article_id", () => {
+  test("200: Responds with an article object with all neccessary properties", () => {
+    return request(app)
+    .get("/api/articles/2")
+    .expect(200)
+    .then(({body : {article}}) => {
+      expect(typeof article.author).toBe("string")
+      expect(typeof article.title).toBe("string")
+      expect(typeof article.article_id).toBe("number")
+      expect(typeof article.body).toBe("string")
+      expect(typeof article.topic).toBe("string")
+      expect(typeof article.created_at).toBe("string")
+      expect(typeof article.votes).toBe("number")
+      expect(typeof article.article_img_url).toBe("string")
+  })
+})
+test("404: Responds with error 404 for an invalid article_id", () => {
+  return request(app)
+  .get("/api/articles/22")
+  .expect(404)
+})
```

Comment:
This should be for a 404 non-existent ID, such as **/api/articles/999999** 
Make sure you write a test for a status 400 - invalid ID **'api/articles/not-an-id'**